app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -85,9 +87,14 @@ class PatientSummaryUpdateTest {
   }
 
   @Test
-  fun `when there are patient summary changes and at least one measurement is present and no diagnosis is recorded, then clicking on back must show diagnosis error`() {
+  fun `when there are patient summary changes and at least one measurement is present and no diagnosis is recorded and diabetes management is enabled, then clicking on back must show diagnosis error`() {
+    val facility = TestData.facility(
+        facilityConfig = FacilityConfig(diabetesManagementEnabled = true)
+    )
```

Comment:
We can create two facilities in the test class:

```kotlin
private val facilityWithDiabetesManagementEnabled = TestData.facility(
  uuid = UUID.fromString("abe86f8e-1828-48fe-afb5-d697b3ce36bb"),
  facilityConfig = FacilityConfig(diabetesManagementEnabled = true)
)

private val facilityWithDiabetesManagementDisabled = TestData.facility(
  uuid = UUID.fromString("abe86f8e-1828-48fe-afb5-d697b3ce36bb"),
  facilityConfig = FacilityConfig(diabetesManagementEnabled = false)
)
```

And then use these in tests as per requirement.